packages/components/src/components/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -24,7 +24,12 @@ export const WithCustomColors = (): JSX.Element => (
   <Box.div display="flex" flexDirection="column" gap="space40">
     <ProgressBar
       backgroundColor="colorAvatarBackgroundPink"
-      indicatorColor="colorAvatarBackgroundGreen"
```

Comment:
I believe that is not necessary or we can add a new story like "overwriting styles"

Reply:
Yes, it's better to create a new story for this case.